### PR TITLE
Improve help page text

### PR DESCRIPTION
**Pull request by paultcochrane**
While reading the Pull Request Club help page I noticed some minor grammatical errors and places where the sentence flow could be improved.  I've implemented these improvements here.  I've split the changes into separate commits so that you can see the changes more easily (I've also wrapped the code in the first commit so that the remaining diffs are easier to read).  The complete set of changes displayed by GitHub unfortunately obscures the textual changes made here, so it is likely better to consider each commit (and its diff) separately.

This PR is intended to be helpful. If you would like anything changed with this PR, please don't hesitate to let me know and I'll update it and resubmit as necessary. :-)

### CODE REVIEW

**Review by kyzn — APPROVED** [review]

File: root/static/html/help.html [hunk 1]
```diff
@@ -139,8 +139,8 @@ <h4>Can you guarantee a user will submit a pull request to my repository?</h4>
   <br>
 
 
-  <h4>How can I know who have received my repositories as their assignments?</h4>
   <p>You can see your assignees in <a href="/history">history</a> page.
+  <h4>How can I know who has received my repositories as their assignments?</h4>
```
Comment (kyzn) [inline root/static/html/help.html:142]: Hi! Was this line made by mistake by any chance? It seems the question now shows up after the answer.

File: root/static/html/help.html [hunk 2]
```diff
@@ -139,8 +139,8 @@ <h4>Can you guarantee a user will submit a pull request to my repository?</h4>
   <br>
 
 
-  <p>You can see your assignees in <a href="/history">history</a> page.
   <h4>How can I know who has received my repositories as their assignments?</h4>
+  <p>You can see your assignees on the <a href="/history">history</a> page.
```
Comment (kyzn) [inline root/static/html/help.html:143]: Ah, I think this reverses the change in previous commit.

**Comment by kyzn** [comment]
Thanks again!

**Comment by kyzn** [comment]
Just to let you know, I took the liberty to remove the change where the question has moved around. I made sure to keep `s/have/has/` change, which to be honest I didn't notice at first :sweat_smile: Thanks again!

File: root/static/html/help.html (hunk 1 above)
Comment (paultcochrane) [reply root/static/html/help.html:142]: Hrm, that's weird. Sorry about that!

**Comment by paultcochrane** [comment]
Thanks for merging :-)  And sorry for the extra work, I try to check all my PRs as carefully as I can, so no idea how that one slipped through. Thanks for the fixup :-)

**Comment by kyzn** [comment]
No worries and thanks again! :rocket:

**Pull request merged**